setup.cfg

```diff
@@ -1,3 +1,3 @@
 [flake8]
 max-line-length = 160
-ignore = E114, E127, E124, E221, W293, E721, W503, F841, F401
+ignore = E114, E127, E124, E221, W293, E721, W503, F841, F401, E202
```

Comment:
Why do you prefer to ignore this error?

Reply:
Good question! :) 

This is the reson:
```
src/perturbopy/test_utils/compare_data/h5.py:54:64: E202 whitespace before '}'
        diff_str = f'{diff:.1e}, {rdiff * 100:.1e}%, {scalar1 = }, {scalar2 = }'
```

flake8 (or PEP) does not want me to put a space before `}` which is totally fine in the plain code. However, for f-strings it does not make sense: this `f'{x = }'` is more stylistically correct than this ` f'{x=}'`. Because for example, in the plain code, this `x=5` would be a PEP violation and this `x = 5` would not.

What do you think?

Reply:
Agree, totally make sense